requirements/dev.txt

```diff
@@ -5,34 +5,33 @@
 #    pip-compile --output-file=requirements/dev.txt requirements/base.in requirements/dev.in
 #
 appdirs==1.4.4            # via fs, virtualenv
-boto3==1.14.15            # via fs-s3fs
-botocore==1.17.15         # via boto3, s3transfer
+boto3==1.16.25            # via fs-s3fs
+botocore==1.19.25         # via boto3, s3transfer
```

Comment:
Looks like we still need a `make upgrade` job for this repo.

Reply:
ran `make upgrade` again.